clojure-mode.el

```diff
@@ -2684,6 +2684,19 @@ lists up."
     (insert sexp)
     (clojure--replace-sexps-with-bindings-and-indent)))
 
+(defun clojure--collect-ns-aliases ()
+  "Collect all aliases in current ns form."
+  (save-excursion
+    (clojure--find-ns-in-direction 'backward)
+    (let ((end (cdr (bounds-of-thing-at-point 'list)))
+          (rgx (rx ":as" (+ space)
+                   (group-n 1 (+ (not (in " ,]\n"))))))
+          (res ()))
+      (while (re-search-forward rgx end 'noerror)
+        (message (match-string 1))
+        (push (cons (point) (match-string-no-properties 1)) res))
+      res)))
```

Comment:
I'm not sure I understand the value of returning cons cells here if the first item in the cons cell is not the name the alias is referring to.

Reply:
Hmm, I wrote this a few months ago and can't remember why I had that in the first place, will simplify

Reply:
👍